tests/integration_tests/conftest.py

```diff
@@ -102,6 +107,29 @@ def setup_image(session_cloud):
     log.info('Done with environment setup')
 
 
+def _collect_logs(instance, node_id):
+    instance.execute(
```

Comment:
Future me wants a docstring describing params, specifically I'd like an example of what node_id value is generally expected to be so I don't have to look it up from the PR subtext.